fisuser-service/src/main/resources/application.yml

```diff
@@ -0,0 +1,36 @@
+camel:
+  springboot:
+    # The Camel context name
+    name: CamelRestSql
+
+# Binding health checks to a different port
+management:
+  port: 8081
+
+# disable all management enpoints except health
+endpoints:
+  enabled: false
+  health:
+    enabled: true
+
+mysql:
+  service:
+    # The name of the service hosting the MySQL database server,
+    # can be customized using the 'MYSQL_SERVICE_NAME' env variable to use a different service
+    name:     mysql
+    # The database to use, can be customized using the 'MYSQL_SERVICE_DATABASE' env variable
```

Comment:
if this be customized, shouldn't it be a parameter placeholder like e.g.: ${DBNAME}

Reply:
this is already customized, I am just reading here the already mentioned parameters here like : ${mysql.service.name}